org.eclipse.xtext.xtext.ui.tests/src/org/eclipse/xtext/xtext/ui/editor/quickfix/XtextGrammarQuickfixProviderTest.java

```diff
@@ -333,7 +333,7 @@ private String grammarGeneratingModel(String... bodyContent) {
 		list.add("grammar org.xtext.example.mydsl.MyDsl with org.eclipse.xtext.common.Terminals");
 		list.add("generate myDsl \"http://www.xtext.org/example/mydsl/MyDsl\"");
 		list.addAll(Arrays.asList(bodyContent));
-		return Strings.concat("\n", list);
+		return Strings.concat(System.lineSeparator(), list);
```

Comment:
Should this be done for the other lines in this file, too, that use `Strings.concat` ?

Reply:
there are more?

Reply:
e.g. line 82

Reply:
good catch. updated